boards/arm/frdm_k64f/Makefile.board

```diff
@@ -4,4 +4,6 @@ DEBUG_SCRIPT = openocd.sh
 OPENOCD_LOAD_CMD = "flash write_image erase ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
 OPENOCD_VERIFY_CMD = "verify_image ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
 
-export FLASH_SCRIPT OPENOCD_LOAD_CMD OPENOCD_VERIFY_CMD
+JLINK_DEVICE = MK64FN1M0xxx12
```

Comment:
The Segger flash tool is Windows only. I've had some success getting 'make flash' to work with pyOCD on k64 and kl25z, but I'm looking into when pyOCD can support kw41 before submitting a PR for it.